lib/clipboard.zsh

```diff
@@ -30,6 +30,17 @@ function clipcopy() {
     else
       cat $file > /dev/clipboard
     fi
+  elif [[ $OSTYPE == linux-android ]]; then
+    if (( $+commands[termux-clipboard-set] )); then
+      if [[ -z $file ]]; then
+        termux-clipboard-set
+      else
+        cat "$file" | termux-clipboard-set
+      fi
+    else
+      print "clipcopy: termux-api not installed: https://github.com/termux/termux-api-package" >&2
```

Comment:
Link should be https://play.google.com/store/apps/details?id=com.termux.api

Reply:
Maybe https://wiki.termux.com/wiki/Termux:API , since the `termux-api` package is also needed and it's clearer reading it in that wiki.